pages/classes/wise.md

```diff
@@ -26,10 +26,169 @@
+
+</section>
+
+<section class="spell">
+
+<header>
+
+#### Purify Elements
+
+<p class="subheading">1st-level Elementalist ritual</p>
+
+</header>
+
+You can sense and purify small amounts of [Close](../../pages/rules/distance.md) natural matter. Food and drink is rendered free of poison and disease, flames burn without smoke, and small animals are healed. Magical effects are immune to this ritual.
+
+</section>
+
+</section>
+
+### Elementalism
+
+<p class="subheading">1st-level Elementalist archetype feature</p>
+
+</header>
+
+You can cast spells. To cast a spell you must expend a spell slot. Each spell takes an [Action](../../pages/combat/actions.md) or  [Bonus Action](../../pages/combat/bonus-actions.md)  to cast. You regain your spell slots when you finish a [Rest](../../pages/rules/rests.md).
+
+Your **Spell Difficulty** is:
+
+<div class="example-roll">
+  8
+  +
+  <span class="bonus"> <span class="caption">Proficiency</span></span>
+  +
+  <span class="bonus"> <span class="caption">Wisdom</span></span>
+</div>
+
+<header>
+
+<section class="spells">
+
+<section class="spell">
+
+<header>
+
+#### Channel Essence [Bonus Action]
+
+<p class="subheading">1st-level Elementalist spell</p>
+
+</header>
+
+A [Nearby](../../pages/rules/distance.md) creature regains **1d4 + Wisdom** [Stamina](../../pages/combat/stamina.md). When you cast this spell using a **Spell Slot** of 2nd level or higher, add an additional **1d4** for each slot level above 1st.
+
+</section>
+
+<section class="spell">
+
+<header>
+
+#### Primordial Strength [Action]
+
+<p class="subheading">1st-level Elementalist spell</p>
+
+</header>
+
+You channel a burst of raw, primordial energy through your body. You or a [Close](../../pages/rules/distance.md) creature regains **1d8 + Wisdom Stamina**. When you cast this spell using a **Spell Slot** of 2nd level or higher, add an additional **1d8** for each slot level above 1st.
+
+</section>
+
+</section>
+
+<header>
+
+### Primordial Form
+
+<p class="subheading">2nd-level Elementalist archetype feature</p>
+
+</header>
+
+You can use your [Action](../../pages/combat/actions.md) to assume an animal form. You use this feature again when you [Rest](../../pages/rules/rests.md).
+
+When you transform you have **10 + your level Stamina**. When you revert this Stamina is lost, and you return to your Stamina before you changed.
+
+When you transform you may pick 2 of the following benefits for your new form:
+
+ * You gain a **Bite or Claw (Melee 1d6, Light) Attack**.
+ * You become tiny and/or can fly (you cannot attack in this form).
+ * You can breath water.
+ * You can use a [Bonus Action](../../pages/combat/bonus-actions.md) to make the [Dash](../../pages/combat/actions.md#dash) [Action](../../pages/combat/actions.md).
+ * You have advantage on all [Perception](../../pages/characters/skills.md#perception/) [Checks]((../../pages/rules/rolling/checks.md).
```

Comment:
Unsure if primordial form is balanced. Given it's rest-restricted, it might be equivalent to 2 eldritch invocations. For example, agonising blast + armor of shadows.

Should we just replace this feature with a couple of eldritch invocations?